docs/compilation.rst

```diff
@@ -28,7 +28,7 @@ Building Python wheel and (optionally) Docker image
 
 Change directory (``cd``) into ``docker`` directory and run ``./build.sh``. If needed, set the following environment variables:
 
-* PYVER - Python version. The default is ``3.6``.
+* PYVER - Python version user to create docker image with DALI installed inside. The default is ``3.6``.
```

Comment:
3.7?

Reply:
also is it PYVER or PYV that is still retained?  There's seemingly a mix of them in this PR.

Reply:
It is used only for docker where image where you get DALI installed and you can run it, not for the build itself. Anyway reverted to 3.6 everywhere.